.github/workflows/pytest.yml

```diff
@@ -9,12 +9,16 @@ on:
 jobs:
   build:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Python Version 3.8 should not be supported and will be removed from this PR. The default version of python for the Raspberry Pi is currently 3.9, thought it would be best to support the default version and updates afterwards until the Raspberry Pi makes a newer version its default